setupext.py

```diff
@@ -168,31 +164,29 @@ def write_cache(local_fn, data):
 options['local_freetype'] = lft or options.get('local_freetype', False)
 
 
-# Define the display functions only if display_status is True.
-if options['display_status']:
-    def print_line(char='='):
-        print(char * 80)
+print_raw = ((lambda *args, **kwargs: None)
+             if '-q' in sys.argv or '--quiet' in sys.argv
+             else print)
```

Comment:
I never liked the practice of assigning a single lambda to a variable. I think this reads better anyway like:
```
if '-q' in sys.argv or '--quiet' in sys.argv:
    def print_raw(*args, **kwargs):
        pass
else:
    print_raw = print
```

At the very least, I would like to see some sort of comment explaining why we need to do this, perhaps "We need to optionally suppress our own output"?

Reply:
sure, you got both the `def` and the comment :)